routes/project/postgres_location.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Clover
+  hash_branch(:project_prefix, "postgres-location") do |r|
+    r.get true do
+      authorize("Postgres:view", @project)
+      {items: vm_families_for_project(@project).map { |l| Serializers::PostgresLocation.serialize_internal(l) }}
```

Comment:
I think this should work:

```suggestion
      {items: Serializers::PostgresLocation.serialize(vm_families_for_project(@project))}
```